### DISCUSSION TITLE
ArcGIS Enterprise Logins with IWA?

### DISCUSSION THREAD

**Discussion by mgress12** [EsriPS/arcgis-assistant-feedback#38] [Q&A]
Does this app support enterprise portals configured with IWA/single sign on? The old ArcGIS online assistant had an option to login with IWA.
I've added and registered the app to Portal. Double checked the redirect URI but not having luck connecting.

I get a prompt for an approval that finds my credentials in Portal. After approving I'm returned to ArcGIS Assistant with only the top menu bar visible and the rest of the screen is grayed out.

Here are the registered app settings:

![App_Reg_Info](https://user-images.githubusercontent.com/77856661/149581222-4ef2d65a-ef8d-4ea4-bc49-2538ce591c6d.PNG)

Thanks!
Matt

**Comment by jpeterson** [comment]
@mgress12 thanks for sharing these details – support for web-tier authentication (IWA and PKI) is on our backlog, so I've created #40 to track our progress on it.
